fastlane/Fastfile

```diff
@@ -238,7 +239,42 @@ end
       project_slug: 'simplenote-macos',
       dsym_path: dSYM_PATH,
     )
-   
+
+  end
+
+  # Temporary lane to automate the process of uploading the .app manually
+  # exported after archiving the Simplenote scheme from Xcode.
+  #
+  # The process is not automated via the build_and_upload_beta lane due to
+  # issues with Sparkle and code signing.
+  lane :upload_app_to_appcenter do | options |
+    app_path = options[:app_path]
+    UI.user_error! "Missing .app path option.\nUsage:\n\tfastlane upload_app_to_appcenter app_path:path/to/.app" if app_path.nil?
+
+    app_path = File.expand_path(app_path)
+    UI.user_error! "Could not find .app at #{app_path}" unless File.exist?(app_path)
+
+    Tempfile.create do |t|
+      release_notes_path = File.path(t)
+      system "pushd ..; ruby ./Scripts/extract_release_notes.rb -k > #{release_notes_path}; popd"
```

Comment:
This is the only logic we might want to save once I'll get the archiving, code signing, and notarization to work for the non-App-Store scheme. It removes the need to copy paste the release notes into App Center, a quite minor but still annoying piece of rote work in the process.